src/Toggle/Toggle.js

```diff
@@ -184,13 +192,15 @@ class Toggle extends Component {
       styles.track,
       this.props.trackStyle,
       this.state.switched && styles.trackWhenSwitched,
+      this.state.switched && this.props.trackSwitchedStyle,
```

Comment:
`trackSwitchedStyle` and `thumbSwitchedStyle ` will end up in the `other` variable. I'm wondering if we shouldn't destructure those properties to prevent this.